tools/code-tools/jcstress.sh

```diff
@@ -19,11 +20,34 @@ function hashArtifacts() {
 function detectJdks() {
   jvm_dir="/usr/lib/jvm/"
   find ${jvm_dir} -maxdepth 1 | sort
+  echo "Available jdks 8 in ${jvm_dir}:"
+  echo "  Note, that you need at at least jdk 11 to build 20240222, tip and 0.16. Listing all jdks anyway."
+  find ${jvm_dir} -maxdepth 1 | sort | grep -e java-1.8.0- -e jdk-1.8.0 -e java-8- -e jdk-8
   echo "Available jdks 11 in ${jvm_dir}:"
   find ${jvm_dir} -maxdepth 1 | sort | grep -e java-11- -e jdk-11
+  echo "Available jdks 17 in ${jvm_dir}:"
+  find ${jvm_dir} -maxdepth 1 | sort | grep -e java-17- -e jdk-17
+  jdk8=$(readlink -f $(find ${jvm_dir} -maxdepth 1 | sort | grep -e java-1.8.0- -e jdk-1.8.0 -e java-8- -e jdk-8 | head -n 1))
   jdk11=$(readlink -f $(find ${jvm_dir} -maxdepth 1 | sort | grep -e java-11- -e jdk-11 | head -n 1))
+  jdk17=$(readlink -f $(find ${jvm_dir} -maxdepth 1 | sort | grep -e java-17- -e jdk-17 | head -n 1))
```

Comment:
Do we still need this?

Reply:
Are we missing Java 21 here as well?

Reply:
What do you mean miss java 21?  The container have jdk8, jdk11, jdk17 and also jdk21 iirc. Jcsress since some 2023 needs at elast jdk11 to build with. I'm checking the hdk versiosn just for case the build goes wrong. I can check the 21 too if you wish.

the upstream is correctlysetting source/version/release javc flags so to build each version by different jdk was just candy on the top.   Now all three jcstresses (latest release, tip, and  "latest used by us" )  are built by  jdk11. If you wish, I have no objections to build tip by soemthign newer. :) ty!

Reply:
It sounds like you only need to check for the existence of 11+ in that case?  It seems over kill to check for each LTS version...